scripts/get-versions.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ueo pipefail
+
+DIR=$(dirname -- "${BASH_SOURCE[0]}")
+
+# Gather a map of the versions from this or another workspace.
+WORKDIR=${1:-.}
+jq -r '.workspaces[]' "$WORKDIR/package.json" | (
+  cd "$WORKDIR"
+  while read PACKAGEDIR; do
+    (
+      cd "$PACKAGEDIR"
```

Comment:
This is not tolerant of wildcards in the workspaces.  I'd prefer using `yarn workspaces info` directly, which would support an idiom like:

```suggestion
cd "$WORKDIR"
yarn workspaces --json info | jq -r '.data | fromjson | .[].location' | \
while read PACKAGEDIR; do
  (
     cd "$PACKAGEDIR"
```